pkg/dnsutil/hostsstore/hostsstore.go

```diff
@@ -56,7 +56,7 @@ func ensureFile(path string) error {
 		return err
 	}
 	f, err := os.OpenFile(path, os.O_CREATE, 0644)
```

Comment:
Why not use `os.WriteFile`?

Reply:
no need to write here, IMO `os.OpenFile` was fine